project/src/main/webapp/events-feed.html

```diff
@@ -99,6 +99,10 @@ <h3>Discover Events Around You</h3>
               </div>
             </div>
         </div>
+        <button type="button" class="close close-btn" aria-label="Close"  
+        onclick="window.location.href='https://event-organizer-step-2020.uc.r.appspot.com/events-feed.html';"> 
```

Comment:
Can you move this to the corresponding JS so scripts stay in that file and we can find them easier?

Reply:
yes done